lib/stack_master/parameter_resolvers/acm_certificate.rb

```diff
@@ -0,0 +1,32 @@
+module StackMaster
+  module ParameterResolvers
+    class AcmCertificate < Resolver
+      CertificateNotFound = Class.new(StandardError)
+
+      def initialize(config, stack_definition)
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def resolve(value)
+        cert_arn = all_certs.map { |c| c.certificate_arn if c.domain_name == value }.compact.first
```

Comment:
Maybe a extract method with this logic (`find_cert_arn_by_domain_name`?)

Reply:
:+1: